README.md

```diff
@@ -25,22 +25,17 @@ Confirm that the following requests work for you
 
 Follow these steps to get Dynamodb running locally
 
-1. `cd CommunityFridgeMapApi/`
-2. **Create a Docker bridge network**
-    1. `docker network create cfm-network`
-    2. `docker run --network cfm-network --name dynamodb -d -p 8000:8000 amazon/dynamodb-local`
-        * If dynamodb container is stopped, start it back up:
-            * ```docker start dynamodb```
-3. **Create Dynamodb tables locally**
-    1. **fridge:** `aws dynamodb create-table --table-name fridge --attribute-definitions AttributeName=id,AttributeType=S --key-schema AttributeName=id,KeyType=HASH --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-    2. **fridge_report:** `aws dynamodb create-table --table-name fridge_report --attribute-definitions AttributeName=fridge_id,AttributeType=S AttributeName=timestamp,AttributeType=N --key-schema AttributeName=fridge_id,KeyType=HASH AttributeName=timestamp,KeyType=RANGE --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-    3. **fridge_history:** `aws dynamodb create-table --table-name fridge_history --attribute-definitions AttributeName=fridge_id,AttributeType=S AttributeName=timestamp,AttributeType=N --key-schema AttributeName=fridge_id,KeyType=HASH AttributeName=timestamp,KeyType=RANGE --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-    4. **tag**: `aws dynamodb create-table --table-name tag --attribute-definitions AttributeName=tag_name,AttributeType=S --key-schema AttributeName=tag_name,KeyType=HASH  --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-4. **Build the functions inside a Docker container**
-    1. `sam build --use-container`
-5. **Load data into your local Dynamodb tables**
+1. **Start a local DynamoDB service**
+    ```sh
+    docker compose up
+    ```
+1. **Create tables**
+    ```sh
+    ./scripts/create_local_dynamodb_tables.sh
```

Comment:
Getting the following error when running this command:

```
./scripts/create_local_dynamodb_tables.sh: line 2: realpath: command not found

Error parsing parameter '--cli-input-json': Unable to load paramfile file:///*.json: [Errno 2] No such file or directory: '/*.json'
Failed to create table from /*.json
```

Reply:
I didn't know `realpath` was not available in some OS. Fixed in d5b9829dc453d2088ac8770e76ba0ac96970e4d2.
It should work when you run ./scripts/create_local_dynamodb_tables.**py**